packages/openspp/test/job.example.js

```diff
@@ -0,0 +1,6 @@
+getServicePoint("Agent 1");
```

Comment:
@josephjclark I hope this is what you need, coz sometimes, words don't have meanings with me 😢 if not, I am sorry. I tried to run this job with command:
```
openfn <path_to_file> -O -ma openspp
```
but it seems that the configuration credentials is not right...

also tried:
```
openfn --help
```
but it seems your openfn-cli command does not support settings for configuration yet...

if this is really what you wanted, can you teach me how to run this in my local machine, using all the credentials I had for ensuring what I coded

Reply:
thanks for staying with me, I truly am ❤️

Reply:
Looks absolutely perfect!

When you run from the cli you'll need to pass state with `-s path/to/state.json`. The state file needs a `configuration` key with the your config in.

I'm typing on mobile right now, I can share more details later if you need it.

Reply:
yep, I tried in my local and it works perfectly

Reply:
Great!